### PR TITLE
follow-up: upgrade to go1.20

### PR DESCRIPTION
**Pull request by fahedouch**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/containerd/nerdctl
 
-go 1.19
```
Comment (AkihiroSuda) [inline go.mod:3]: This is intended to be the minimum supported version

Comment (fahedouch) [reply go.mod:3]: Why ? is there sthg blocking on containerd side ?

Comment (fahedouch) [reply go.mod:3]: Sorry I miss understood your comment! ok but is it the right place to put  the minimum supported version ? the `go.mod` ?

Comment (fahedouch) [reply go.mod:3]: SGTM `A list of minimum versions of other modules required by the current module.` from official [golang doc](https://go.dev/doc/modules/gomod-ref)

Comment (fahedouch) [reply go.mod:3]: Thanks for your feedback

**Pull request closed without merging**